streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -1223,14 +1237,24 @@ public void onPartitionsAssigned(Collection<TopicPartition> assignment) {
                 rebalanceException = t;
                 throw t;
             } finally {
-                log.debug("{} partition assignment took {} ms", logPrefix, time.milliseconds() - start);
+                log.debug("{} partition assignment took {} ms.\n" +
```

Comment:
i'm wondering if this one should be at `info` level? It would make it easier to find out when a thread has finished rebalancing. Right now it is a case of looking for some other message that happens during processing. Seeing as rebalancing shouldn't, in theory, happy that often, i think info would be ok.